app/views/oauth_applications/app_owner_application.html.haml

```diff
@@ -0,0 +1,21 @@
+- content_for :title do
+  =t :apply_for_permissions_to_create_inat_third_party_applications_title
```

Comment:
This text needs to be added to en.yml, or just use `apply_to_create_third_party_applications_title`